docs/documentation/tests/src/test/resources/ignored-link-redirects

```diff
@@ -15,4 +15,5 @@ https://access.redhat.com/documentation/en/red-hat-jboss-middleware-for-openshif
 https://access.redhat.com/login?redirectTo=https://access.redhat.com/terms-based-registry
 https://sso.redhat.com/auth/*
 REGEX:https://access.redhat.com/documentation/.*/index
+https://nodejs.org/en
```

Comment:
Why do we need to add this exception?

Reply:
I think this was due to an error during the documentation build that showed up when I did a full build.
I can try to remove it again.

Reply:
Yeah, if we can avoid it let's.